bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/handler/DanfossAirUnitHandler.java

```diff
@@ -62,6 +64,7 @@
     private @Nullable ScheduledFuture<?> pollingJob;
     private @Nullable DanfossAirUnitCommunicationController communicationController;
     private @Nullable DanfossAirUnit airUnit;
+    private boolean propertiesInitializedSuccessfully = false;
```

Comment:
Would it make sense to also set propertiesInitializedSuccessfully to false in dispose() ?

Reply:
I don't think so. The properties are static (unit name and serial number), so once initialized there will be no need for doing that again.

Reply:
Yes, considering the kind of properties, you're right.